services/server/server.go

```diff
@@ -600,6 +603,16 @@ func (s *Server) ChangeSettings(ctx context.Context, req *serverpb.ChangeSetting
 		return nil, err
 	}
 
+	if req.EnableStt {
```

Comment:
Can we check that before STT was disabled? Because frontend always passes all fields.

Reply:
We can get old settings before applying an update, and then compare STT status changes. Is it worth it?

Reply:
I think yes, because when user updates some other setting (e.g. metrics resolution) he don't expect that force check downloads will be executed

Reply:
Done, also I fixed https://jira.percona.com/browse/PMM-7343 in this PR